saleor/graphql/webhook/mutations/event_delivery_retry.py

```diff
@@ -25,10 +29,19 @@ class Meta:
 
     @classmethod
     def perform_mutation(cls, _root, info: ResolveInfo, **data):
+        apps = App.objects.filter(removed_at__isnull=True).values("pk")
+        webhook = Webhook.objects.filter(
+            Exists(apps.filter(id=OuterRef("app_id")))
+        ).values("pk")
+        deliveries = models.EventDelivery.objects.filter(
+            Exists(webhook.filter(id=OuterRef("webhook_id")))
+        )
```

Comment:
I think I saw getting deliveries is used a few times in the code, how about a helper function for that?

Reply:
I don't think that such util will be helpful. We don't use a similar approach in e.g. products ORM queries. 

In my opinion, it's easier to look at 3 simple QS instead of looking inside function.